### DISCUSSION TITLE
Unterstützung für e-Paper Display machbar?

### DISCUSSION THREAD

**Discussion by MrHonk** [tbnobody/OpenDTU#819] [General]
Hallo,

wäre es möglich, das "1.54 inch e-Paper Display" von Waveshare in die Liste der unterstützten Displays mit aufzunehmen?
Ich würde es ja selber versuchen, leider fehlt mir dazu das nötige Fachwissen. Aber vielleicht ist die Implementierung ja mit Hilfe der Dokumentation des Herstellers für erfahrene Programmierer nur eine Kleinigkeit?

Link zur Developer-Seite des Herstellers: https://www.waveshare.com/wiki/1.54inch_e-Paper_Module

Schöne Grüße
Thomas

**Comment by jlf-cleaner** [comment]
Das würde mich auch sehr interessieren - habe mir das Display bestellt, weil ich es mit Ahoy nutzen wollte (dort wird es wohl unterstützt), aber bin dann doch bei OpenDTU gelandet.

**Comment by alekob** [comment]
Wäre auch daran interessiert, damit wäre das Fusion-Board mit dem ePaper Gehäuse perfekt. Falls eine Implementierung schon auf einem Dev-Branch liegt, würde ich auch gerne testen?!

**Comment by Wookbert** [comment]
Yes, please... Support for the widely known 1.54" SPI ePaper Display as e.g. available from WaveShare and GoodDisplay.

**Reply by mtavenrath** [reply]
There is support for the WaveShare 1.54" e-Paper Display in this PR: https://github.com/tbnobody/OpenDTU/pull/1703.

It's based on the great work by @dAjaY85  and have been changed so that rebasing in the future will be more easy to keep it alive until it hopefully gets merged in the future.
